pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
 	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	ClusterinformationV1beta1 "github.com/vmware-tanzu/antrea/pkg/apis/clusterinformation/v1beta1"
```

Comment:
package names and alias use lowercase by convention, `clusterinformationv1beta1`

Reply:
Updated.